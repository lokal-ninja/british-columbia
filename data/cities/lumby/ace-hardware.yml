id: 691108177
type: way
latitude: 50.248571500000004
longitude: -118.9664331
postcode: ""
city: Lumby
street: "Shuswap Avenue"
housenumber: 2060
phone: ""
opening_hours: ""
website: ""
