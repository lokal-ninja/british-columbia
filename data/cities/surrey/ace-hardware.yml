id: 6919611296
type: node
latitude: 49.10371
longitude: -122.73284
postcode: ""
city: Surrey
street: ""
housenumber: 
phone: ""
opening_hours: "Mo-Su 09:00-17:00"
website: ""