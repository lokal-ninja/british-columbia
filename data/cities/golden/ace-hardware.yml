id: 4517783793
type: node
latitude: 51.29562
longitude: -116.96269
postcode: "V0A 1H0"
city: Golden
street: "10th Avenue South"
housenumber: 914
phone: "+1-250-344-5714"
opening_hours: ""
website: "http://www.truhardware.ca"